.github/workflows/docker-publish.yml

```diff
@@ -163,7 +163,7 @@ jobs:
           namespace: 'discord-bots'
           secret-name: 'tildy'
           secret-type: 'Opaque'
-          string-data: '{"DISCORD_TOKEN":"${{ secrets.DISCORD_TOKEN }}", "TLD_AIRTABLE_KEY": "${{ secrets.AIRTABLE_KEY }}", "TLDBOTTO_CLICKUP_TOKEN": "${{ secrets.CLICKUP_TOKEN }}", "SNAILEDIT_AIRTABLE_KEY": "${{ secrets.SNAILEDIT_AIRTABLE_KEY }}", "APP_STORE_CONNECT_API_KEYS": "${{ secrets.APP_STORE_CONNECT_API_KEYS }}"}'
```

Comment:
Do we not need the rest of these tokens anymore?

Reply:
> Do we not need the rest of these tokens anymore?

We don't. The annotations I added yesterday on [deployment.tpl.yaml](https://github.com/Lovely-Development-Team/TLDBotto/commit/bf63c9467e253d50f92e48259a41dd42f1a250be) cause the 1Password Operator to create the `tildy-credentials` secret for us.

Reply:
That makes sense, thanks